core/src/main/java/dev/hypera/scaffolding/Scaffolding.java

```diff
@@ -97,7 +92,6 @@ private Scaffolding() {}
     /**
      * @param file      the {@link File} to read from
      * @param schematic the {@link Schematic} to load the data into
-     *
```

Comment:
This line shouldn't be removed.

Reply:
Whole code is auto-formatted by intelliJ, why should be it that way like you did it? Its more difficult to maintain a custom layout other than using just intelliJ`s one

Reply:
That's a really backwards way of thinking about that. We use our own code style for readability and maintainability. All of our projects respect that and this project is no different.

Reply:
Alright i understand you both now, is there a way to use your format in IntelliJ without memorizing it?